high_performance_computing/computer_simulations/03_towards_future.md

```diff
@@ -0,0 +1,185 @@
+
+- which discipline does the case study belong to?
+- is there any social or economic benefit to it?
+- what are the main reasons for using supercomputers to study this problem?
+- in your opinion, what is the most surprising or interesting aspect of this case study?
+- are there any similarities between the case study you picked and the three case studies we have discussed in this week? Why do you think that is?
+
+---
+
+![Very tall building](images/verne-ho-0LAJfSNa-xQ-unsplash.jpg)
+*Image courtesy of [Verne Ho](https://unsplash.com/@verneho) from [Unsplash](https://unsplash.com)*
+
+## Future of Supercomputing - the Exascale
+
+So what does the future of supercomputing look like? Much of the current research and development in HPC is focused on Exascale computing.
+
+For HPC architectures, this can be taken to mean working towards a system with a floating point performance of at least 1 Exaflop/s (i.e. 1018 or a million million million floating point calculations per second).
+
+In 2016, the US government announced the Exascale Computing Project, which aims to have its first supercomputer operating at 1 Exaflop/s or more in production by 2021. If past trends in the Top500 list had been followed, then a 1 Exaflop/s system would have been expected in 2018 – the fact that this date will be missed by at least 2 or 3 years is a measure of the technical challenges involved, both in the hardware and software.
+
+As of 2022 exascale has only been achieved by one system [https://www.top500.org/system/180047/](Frontier) based at Oak Ridge National Laboratory. Although at some level this is just an arbitrary number, it has become a significant technological (and political) milestone.
+
+Some of the main barriers to building a useful and economically viable Exascale machine are:
+
+### Hardware speeds
+
+Since around 2006, there has been little significant increase in the clock frequency of processors. The only way to extract more performance from computers has been through more parallelism, by having more cores per chip, and by making each cores capable of more floating point operations per second. Without a fundamental move away from the current silicon technology, there is no real prospect of significantly higher clock frequencies in the next 5-10 years. There is also not much prospect of reducing network latencies by very much either. On the plus side, new memory designs such as 3D stacked memory, do promise some increases in memory bandwidth.
+
+### Energy consumption
+
+If we were to build a 1 Exaflop/s computer today, using standard Intel Xeon processors, then it would consume around 400 megawatts of power: that’s enough electricity for 700,000 households, or about 1% of the UK’s entire electricity generating capacity! That’s not only hugely expensive, but it would require a big investment in engineering infrastructure, and would be politically challenging from a carbon footprint point of view.
```

Comment:
```suggestion
If we were to build a 1 Exaflop/s computer today, using standard Intel Xeon processors, then it would consume around 400 megawatts of power: that’s enough electricity for 700,000 households, or about 1% of the UK’s entire electricity generating capacity!
That’s not only hugely expensive, but it would require a big investment in engineering infrastructure, and would be politically challenging from a carbon footprint point of view.
There are also plans to use waste heat from supercomputers as heating for homes, or for direct energy recovery which could substantially decrease the cost and environmental impact of running these scales of machines.
```